File: LeetCode/C#/100-199/136.cs
using System;

namespace LeetCode136
{
    public class Solution
    {
        public int SingleNumber(int[] nums)
        {
            int ans = 0;
            foreach (int num in nums)
            {
                ans ^= num;
            }

            return ans;
        }
    }
    class Program
    {
        static void Main(string[] args)
        {
            int[] nums = new int[5]{1,2,4,4,2};
            Console.WriteLine(new Solution().SingleNumber(nums));
            Console.Read();
        }
    }
}

File: LeetCode/C++/100-199/136.cpp
#include <iostream>
#include <vector>
using namespace std;
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        int ans = 0;
        for(int i = 0 ; i < nums.size();++i){
            ans ^= nums[i];
        }
        
        return ans;
    }
};

int main() {
	
	vector<int> nums{1,2,4,4,2};
	Solution sol;
	cout << sol.singleNumber(nums) << endl;
	return 0;
}